analyzer/runtime/extract.go

```diff
@@ -897,6 +917,7 @@ func extractEvents(blockData *BlockData, relatedAccountAddresses map[apiTypes.Ad
 						pt.Mutated = true
 					}
 					registerNFTExist(blockData.PossibleNFTs, eventAddr, tokenID)
+					registerNFTTransfer(blockData.PossibleNFTs, eventAddr, tokenID, toAddr)
```

Comment:
Afaik ERC-721 allows transferring nft to yourself (e.g. from address=to address). Is it ok that num-transfers is increased in that case as well?

Reply:
yeah sure. it doesn't seem like a high obstacle if for example people have to transfer it between two of their own accounts instead

Reply:
ok tentatively on the mint and burn not counting in num_transfers

Reply:
ugh I just saw this other comment from the fungible tokens side

> Mints, burns, and zero-value transfers all count as transfers.

so I'm going to preserve consistency with that instead

Reply:
and I'm opposed to fussing with the abstraction in the other suggestions about inlining registerNFTTransfer and eliminating registerNFTExist

Reply:
Thanks for finding the prior "art" on countin mints/burns. Might be worth mentioning in a comment in our codebase.

Tiny refactor/abstraction: sure, sg.